README.md

```diff
@@ -110,106 +147,31 @@ LicenseChecker supports the following licenses:
    @main
    struct LicenseCheckerCommand: CommandPlugin {
        func performCommand(context: PluginContext, arguments: [String]) async throws {
-           let licenseCheckerPath = try context.tool(named: "license-checker").path.string
-           let rswiftURL = URL(fileURLWithPath: licenseCheckerPath, isDirectory: false)
+           let licenseCheckerPath = try context.tool(named: "license-checker").path
+           let commandURL = URL(fileURLWithPath: licenseCheckerPath.string, isDirectory: false)
    
-           let process = try Process.run(rswiftURL, arguments: arguments)
+           let process = try Process.run(commandURL, arguments: arguments)
            process.waitUntilExit()
    
-           if process.terminationReason != .exit || process.terminationStatus != 0 {
-               Diagnostics.error("license-checker errors detected.")
+           guard process.terminationReason == .exit else {
+               Diagnostics.error("Termination Other Than Exit")
+               return
+           }
+           guard process.terminationStatus == EXIT_SUCCESS else {
+               Diagnostics.error("Command Failed")
+               return
            }
        }
    }
    ```
 
 3. Add a Run Script in BuildPhases
-
+   
```

Comment:
nits: Unnecessary diffs.
```suggestion

```

Reply:
This is an automatic fix by VSCode's Prettier.

Reply:
I feel it's a bit unnatural, but I understand.

Reply:
Maybe I'm wrong, but this is definitely strange. I accept the commit suggestion.